qutebrowser/browser/mouse.py

```diff
@@ -57,7 +57,25 @@ def eventFilter(self, obj, event):
 
             if qtutils.version_check('5.11', compiled=False, exact=True):
                 # WORKAROUND for https://bugreports.qt.io/browse/QTBUG-68076
-                QTimer.singleShot(0, self._widget.setFocus)
+                try:
+                    win_id = self._widget._win_id
+                    PASSTHROUGH_MODES = [usertypes.KeyMode.command,
+                                        usertypes.KeyMode.prompt,
+                                        usertypes.KeyMode.yesno]
+                    if modeman.instance(win_id).mode not in PASSTHROUGH_MODES:
+                        tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                                    window=win_id)
+                        current_index = tabbed_browser.widget.currentIndex()
+                        try:
+                            widget_index = tabbed_browser.widget.indexOf(self._widget.parent())
+                        except RuntimeError:
+                            widget_index = -1
+                        if current_index == widget_index:
```

Comment:
I really hate this method of figuring out if a tab is currently in the background, is there a clearer way that I missed?